e2e/tests-dfx/update_settings.bash

```diff
@@ -75,40 +123,40 @@ teardown() {
     ALICE_WALLET=$(dfx --identity alice identity get-wallet)
     BOB_WALLET=$(dfx --identity bob identity get-wallet)
 
-    dfx canister create hello
+    dfx canister --wallet "${ALICE_WALLET}" create hello
```

Comment:
same note as previous test: Let's leave this test as is with respect to these two ` canister create|install` lines, and if necessary, make a second test that creates/installs using `--wallet`.

Reply:
Does there need to be a second test here? I set it up so that for each of the actions 'set controller' and 'set multiple controllers', there's three versions: without wallet, with wallet, and with wallet version 0.7.2.

Reply:
I was thinking it would be good to have a version of the test that uses the most-default parameters (`dfx canister create hello`), with wallet 0.7.2, which is what the version of the test to the left does.  What do you think?

Reply:
There can't be a 'without wallet, but with wallet 0.7.2' test because wallet 0.7.2 doesn't set the caller as another controller. Unless update-settings is used, but then all you're testing is that update-settings successfully nullifies the effect of the wallet on canister creation.